### PR TITLE
Add readiness and liveness probes

### PR DESCRIPTION
**Pull request by andreaTP**

### CODE REVIEW

File: akka-operator/values.yaml [hunk 1]
```diff
@@ -42,6 +42,22 @@ resources:
...
+  initialDelaySeconds: 20
+  failureThreshold: 10
+
+livenessProbe:
+  httpGet:
+    port: 8558
+    path: "/alive"
+  periodSeconds: 10
+  initialDelaySeconds: 20
+  failureThreshold: 10
```
Comment (patriknw) [inline akka-operator/values.yaml]: I think we can have less threshold here. This means that we wouldn't restart until after 10x10 = 100 seconds?
Maybe 5 ?

Comment (andreaTP) [reply akka-operator/values.yaml]: you are right, even lower, I went for 3 !

**Comment by andreaTP** [comment]
Tested (and fixed 🙂  ), ready for final review! cc. @seglo @chbatey

**Review by patriknw — APPROVED** [review]
LGTM

**Pull request merged**